Tests/ExampleAnalyzers/ExampleAnalyzer2/ExampleAnalyzer2/CodeFixProvider.cs

```diff
@@ -0,0 +1,79 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="CodeFixProvider.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Rename;
+using Microsoft.CodeAnalysis.Text;
+
+namespace ExampleAnalyzer2
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(ExampleAnalyzer2CodeFixProvider)), Shared]
```

Comment:
I assume you haven't written this and it is not for review? It may be worth adding a comment on the purpose of this analyser.